taxi/urls.py

```diff
@@ -1,9 +1,23 @@
 from django.urls import path
 
-from .views import index
+from .views import index, \
+    ManufacturerListView, \
+    CarListView, \
+    car_detail_view, \
+    DriverListView, \
+    driver_detail_view
```

Comment:
```suggestion
from .views import index, (
    ManufacturerListView, 
    CarListView, 
    car_detail_view, 
    DriverListView, 
    driver_detail_view
)
```
Use brackets instead of \